src/Files.App/Helpers/GitHelpers.cs

```diff
@@ -156,6 +197,79 @@ public static bool ValidateBranchNameForRepository(string branchName, string rep
 				branch.FriendlyName.Equals(branchName, StringComparison.OrdinalIgnoreCase));
 		}
 
+		public static void FetchOrigin(string? repositoryPath)
+		{
+			Analytics.TrackEvent("Triggered git fetch");
```

Comment:
This event is already tracked from the git fetch action (actions are automatically tracked) so it can be removed.